app/schemas/federation_entities.json

```diff
@@ -0,0 +1,366 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://diasporafoundation.org/schemas/federation_entities-1.0.0.json",
```

Comment:
Sounds good to me. What do you think about also put the schemas at https://diaspora.github.io/diaspora_federation/schemas (at least for now while it is managed inside this gem)?